### PR TITLE
My batis

### PR DESCRIPTION
**Pull request by Paw3lUlita**

### CODE REVIEW

File: src/main/java/com/solvd/services/AddressService.java [hunk 1]
```diff
@@ -1,24 +1,86 @@
...
+    public Address getById(Long id) {
+        Address address;
+        try (SqlSession sqlSession = sqlSessionFactory.openSession()) {
+            IAddressDAO addressDAO = sqlSession.getMapper(IAddressDAO.class);
+            address = addressDAO.getEntityById(id);
+        }
+        return address;
+    }
+
+    public Address saveAddress(Address address) {
```
Comment (vboltrukanis) [inline src/main/java/com/solvd/services/AddressService.java:35]: Basically add actions should be done like this:
You create an object in java (without id, bcs you don't know which id's are already taken in db)
use service to save your object
in database an id value is assigned
you receive entity from a database with id
you update id in service object
return object with id from service method

File: src/main/resources/myBatis_config.xml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE configuration
+        PUBLIC "-//mybatis.org//DTD Config 3.0//EN"
+        "http://mybatis.org/dtd/mybatis-3-config.dtd">
+<configuration>
+    <properties resource="application.properties"/>
```
Comment (vboltrukanis) [inline src/main/resources/myBatis_config.xml:6]: Push properties file to github with the following format:
```
db.driver=
db.url=
...
```
In this case you don't reveal any credentials, but you will show exactly where this file should be saved, which properties should be assigned.
Also driver and url are not secret, so you can add them, username and password are secret, they should not be here.
Also after you pushed basic prop file, you can add it to gitignore, so you will not worry to accidentally push updated version to github.

**Review by vboltrukanis — CHANGES_REQUESTED** [review]

File: src/main/java/com/solvd/services/AddressService.java (hunk 1 above)
Comment (Paw3lUlita) [reply src/main/java/com/solvd/services/AddressService.java:35]: Yes, but here MyBatis returns entity with generated id. I've checked it already in my IDE. I saw similar solution in  Dimitrys demo repo.

**Review by vboltrukanis — APPROVED** [review]
lgtm

**Pull request merged**